hw/ip/prim/dv/prim_prince/crypto_dpi_prince/crypto_dpi_prince.c

```diff
@@ -0,0 +1,24 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "prince_ref.h"
+#include "svdpi.h"
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
```

Comment:
Please add a 

```
#ifdef __cplusplus
extern "C" {
#endif
```

block around this file.

(Verilator compiles DPI modules (for a variety of reasons incorrectly) in C++ mode; this block helps to prevent name mangling to get in the way.)

(Optional) I would add a crypto_dpi_prince.h file, and remove the `extern` declarations.

Reply:
Sure, will add the extern block, thanks for the input!